FrontEnd/src/components/adminPage/detail-view/UserDetail.jsx

```diff
@@ -1,64 +1,117 @@
+import { useState } from 'react';
+import { useNavigate } from 'react-router';
 import css from './UserDetail.module.css';
-import Checkbox from '../../checkbox/Checkbox';
-import {useState} from 'react';
 import DeleteModal from './DeleteModal';
-
-const FIELDS = [
-    {
-        id: 'f1',
-        label: 'Прізвище',
-        form_name: 'lastname',
-    },
-    {
-        id: 'f2',
-        label: 'Ім\'я',
-        form_name: 'name',
-    },
-    {
-        id: 'f3',
-        label: 'Електронна пошта',
-        form_name: 'email',
-    },
-    {
-        id: 'f4',
-        label: 'Номер телефону',
-        form_name: 'phone_number',
-    }
-];
-
+import axios from 'axios';
+import useSWR from 'swr';
 
 function UserDetail() {
     const [deleteModalActive, setDeleteModalActive] = useState(false);
+    const [user, setUser] = useState([]);
+    const userId = usePathUserId();
+    const [updateSuccess, setUpdateSuccess] = useState(false);
+    const url = `${process.env.REACT_APP_BASE_API_URL}/api/admin/users/${userId}/`;
+    const navigate = useNavigate();
 
-    const handleDeleteClick = () => {
-        setDeleteModalActive(true);
+    const fetcher = url => axios.get(url).then(res => res.data);
+    const { data, error, isValidating: loading } = useSWR(url, fetcher);
+    if (data && !Object.keys(user).length) {
+        setUser(data);
+    }
+
+    const handleSaveChanges = async () => {
+        const response = await axios.put(
+            url,
+            {
+                name: user.name,
+                surname: user.surname,
+                email: user.email,
+                is_active: user.is_active,
+                is_staff: user.is_staff,
+                is_superuser: user.is_superuser
+            },
+        );
+        if (response.status !== 200) {
+            throw new Error(`HTTP error! Status: ${response.status}`);
+        }
+        setUpdateSuccess(true);
+        setTimeout(() => setUpdateSuccess(false), 3000);
```

Comment:
Would it be better to show a toast message?

Reply:
It was a simple solution to display the status.